apps/frontend/site/src/lib/styles/global.css

```
@@ -363,3 +363,25 @@
     }
   }
 }
+
+/* ── Font metric overrides — reduce CLS during web font swap ── */
+/* These @font-face rules merge with Fontsource declarations to add size-adjust. */
+/* Inter: match system-ui metrics at 400 weight (body text) */
+@font-face {
+  font-family: "Inter";
```

Stylelint (17.14.1)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)